cli/tests/cli-reference@.md.snap

```diff
@@ -1811,6 +1813,23 @@ Show commit description and changes in a revision
 
 
 
+## `jj simplify`
+
+Simplify parent edges for the specified revision(s).
+
+Removes any and all redundant parent edges going into any of the specified revisions. By definition, this has no effect on any commit contents and should not cause any working copy changes or conflicts.
```

Comment:
It was not clear to me what a redundant parent edge was before reading the code and playing with the code. Maybe a textual example would be good ("For example, a commit with multiple parents descending from one another will only depend on its closest ancestors." or something like that, even though it is clear what "closest ancestors" mean here).

Reply:
Added a paragraph here.